lib/Terrain.h

```diff
@@ -42,6 +42,11 @@ class DLL_LINKAGE BattleField
 	{
 		h & name;
 	}
+
+	bool isSpecial() const
+	{
+		return name.find('_') >= 0; // hack for special battlefields, move to JSON
```

Comment:
Are you going to have parameter instead?

Reply:
yes, already done